xcm/xcm-executor/src/lib.rs

```diff
@@ -24,7 +24,7 @@ use frame_support::{
 use parity_scale_codec::{Decode, Encode};
 use sp_core::defer;
 use sp_io::hashing::blake2_128;
-use sp_std::{marker::PhantomData, prelude::*};
+use sp_std::{marker::PhantomData, prelude::*, vec};
```

Comment:
This import is relevant for working around the following error which happens when running `cargo publish` on a crate that depends on `xcm-executor`

```
error: cannot find macro `vec` in this scope
  --> /usr/local/cargo/registry/src/-c44c4657aebc2992/xcm-executor-0.9.33/src/lib.rs:444:43
	error: cannot find macro `vec` in this scope
    |
444 |                 Config::XcmSender::send_xcm(dest, Xcm(vec![instruction])).map_err(Into::into)
    |                                                       ^^^
    |
    = note: consider importing one of these items:
            parity_scale_codec::alloc::vec
            sp_std::vec
```